src/sentry/static/sentry/app/views/settings/components/forms/fieldFromConfig.jsx

```diff
@@ -69,6 +73,8 @@ export default class FieldFromConfig extends React.Component {
     };
 
     switch (field.type) {
+      case 'separator':
```

Comment:
omg we've come full circle and are now implementing windows forms.

Reply:
Amazing times we live in.